seatunnel-core/seatunnel-core-spark/src/main/java/org/apache/seatunnel/SparkStarter.java

```diff
@@ -230,6 +232,18 @@ protected List<String> buildFinal() {
         appendFiles(commands, this.files);
         appendSparkConf(commands, this.sparkConf);
         appendAppJar(commands);
+        if (this.commandArgs.getDeployMode().getName() == "cluster") {
```

Comment:
Create a method to cover the logic maybe better?

Reply:
Thank you for your advice. I'll change it

Reply:
can you help me review the code and if ok approve that